integration-test/common/src/test/java/org/apache/uniffle/test/QuorumTest.java

```diff
@@ -316,6 +316,76 @@ public void rpcFailedTest() throws Exception {
     assertEquals(readClient.readShuffleBlockData(), null);
   }
 
+  @Test
+  public void testRpcRetryLogic() {
```

Comment:
Could you move the related retry method to another class?

Reply:
To what class, a new test class? What should the name of the new class be?

Reply:
Maybe `ClientRetryTest.java`.

Reply:
OK.